### DISCUSSION TITLE
Nested "include: where" query

### DISCUSSION THREAD

**Discussion by taylorhlabs** [prisma/prisma#5995] [Q&A] [answered]
In the query below, I need to only include resources that are not deleted but I can't get the syntax right. Is it possible to have a `where` in a nested `include` where I have the comment? 
```
    const boardQuery = await prisma.board.findFirst({
      where: {
        slug: slug,
        userId: token.uid
      },
      include: {
        boardSections: {
          include: {
            boardSectionResources: {
              include: {
                resource: {
                  // where: { deleted: false }, // COMMENTED OUT BECAUSE BREAKS MY QUERY
                  include: {
                    resourceTags: { include: { tag: true } }
                  }
                }
              }
            }
          },
          orderBy: [{ order: 'asc' }]
        }
      }
    });
```

**Comment by tarazena** [comment, marked as answer]
@taylorhlabs can you post the Prisma Schema? is there a `deleted` column in `resource` table?

**Reply by taylorhlabs** [reply]
Hey @tarazena! Yes, there is a `deleted` column in the `resource` table.

**Reply by taylorhlabs** [reply]
Note that Prisma doesn't seem to allow the `where` clause as I have it. It seems to be a syntax error on my part. Here's the error:
```
Unknown arg `where` in include.boardSections.include.boardSectionResources.include.resource.where for type Resource. Did you mean `select`?
```

**Reply by taylorhlabs** [reply]
As I was about to copy/paste my schema, I took another stab at it and realized I had my `where` in the wrong spot. Thanks @tarazena for hinting at the schema 🙏

Here is working version:
```
    const boardQuery = await prisma.board.findFirst({
      where: {
        slug: slug,
        userId: token.uid
      },
      include: {
        boardSections: {
          include: {
            boardSectionResources: {
              where: { resource: { deleted: false } },
              include: {
                resource: {
                  include: {
                    resourceTags: { include: { tag: true } }
                  }
                }
              }
            }
          },
          orderBy: [{ order: 'asc' }]
        }
      }
    });
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
